docs/self-managed/modeler/web-modeler/configuration.md

```diff
@@ -0,0 +1,121 @@
+
+#### WebSocket
+
+The `webapp` component sends certain events (e.g. "user opened diagram", "user left diagram") to the [WebSocket](#configuration-of-the-websocket-component) server and reacts to events triggered by other users.
+
+| Environment variable      | Description                                                                                                                                   | Example value        | Default value |
+| ------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | -------------------- | ------------- |
+| `PUSHER_HOST`             | [Internal](#notes-on-host-names-and-port-numbers) host name of the WebSocket server.                                                          | `modeler-websockets` | -             |
+| `PUSHER_PORT`             | [Internal](#notes-on-host-names-and-port-numbers) port number of the WebSocket server.                                                        | `8060`               | -             |
+| `PUSHER_APP_ID`           | _must be the same as_ [`PUSHER_APP_ID`](#configuration-of-the-websocket-component)                                                            | `web-modeler`        | -             |
+| `PUSHER_KEY`              | _must be the same as_ [`PUSHER_APP_KEY`](#configuration-of-the-websocket-component)                                                           | \*\*\*               | -             |
+| `PUSHER_SECRET`           | _must be the same as_ [`PUSHER_APP_SECRET`](#configuration-of-the-websocket-component)                                                        | \*\*\*               | -             |
+| `CLIENT_PUSHER_HOST`      | [External](#notes-on-host-names-and-port-numbers) host name on which the Web Modeler client accesses the WebSocket server from the browser.   | `ws.example.com`     | -             |
+| `CLIENT_PUSHER_PORT`      | [External](#notes-on-host-names-and-port-numbers) port number on which the Web Modeler client accesses the WebSocket server from the browser. | `443`                | -             |
+| `CLIENT_PUSHER_KEY`       | _must be the same as_ [`PUSHER_APP_KEY`](#configuration-of-the-websocket-component)                                                           | \*\*\*               | -             |
+| `CLIENT_PUSHER_FORCE_TLS` | Enable TLS encryption for WebSocket connections initiated by the browser.                                                                     | `true`               | `false`       |
+
+### Configuration of the `websocket` component
+
+The WebSocket server shipped with Web Modeler Self-Managed is a replacement for the third-party service [Pusher](https://pusher.com) (used by Web Modeler SaaS), implementing the same concepts and APIs. It is used to exchange events between different Web Modeler users to support collaborative modeling.
```

Comment:
Is this information useful for users? I guess you've written it to explain why the vars still are named "pusher". I think it requires another sentence upfront to describe it better. Such as 

> Web Modeler Self-Managed supports collaborative editing. It ships with a WebSocket implementation that follows the same concepts and APIs as [Pusher](https://pusher.com) for standardization reasons. Pusher is used in Web Modeler SaaS. Hence, the environment variables are named the same, but you don't need to point to a running pusher instance.

Reply:
But tbh, for the GA we should introduce an alias for these vars and ditch the "random" secrets that the user does not care about

Reply:
For the record (@christian-konrad, @CatalinaMoisuc): If they use Helm, a random secret will be automatically generated and they don't need to care about it.